src/ucp/proto/proto_multi.inl

```diff
@@ -186,7 +186,7 @@ ucp_proto_multi_bcopy_progress(ucp_request_t *req,
 
 static UCS_F_ALWAYS_INLINE ucs_status_t ucp_proto_multi_zcopy_progress(
         ucp_request_t *req, const ucp_proto_multi_priv_t *mpriv,
-        ucp_proto_init_cb_t init_func, unsigned uct_mem_flags, unsigned dt_mask,
+        unsigned uct_mem_flags, unsigned dt_mask,
```

Comment:
instead of changing init funcs callback, can we pass ucp_eager_am_zcopy_pack_user_header as 'init_func' callback, so this function will call ucp_proto_request_zcopy_init directly and then init_func like before?